doc: release-notes-latest: update ff720cd9b343 synchronization sha (#3817)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-latest.rst

MCUboot
=======

> The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``3f49b5abf3``, plus some |NCS| specific additions. |NCS| specific mcuboot glue code has been moved to :file:`ncs/nrf/modules/mcuboot`.

Reviewer: ```suggestion
The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``3f49b5abf3``, plus some |NCS| specific additions.

The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.
```

Reviewer: This must also be updated in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/readme-ncs.html @tejlmand

Reviewer 2: yep, ready here: https://github.com/nrfconnect/sdk-nrf/pull/3833

======

.. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.

> The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``ff720cd9b343``, plus some |NCS| specific additions.

Reviewer: Need to keep the blank line here - that's why the doc build fails.
